density/density.py

```diff
@@ -213,7 +214,8 @@ def get_day_group_data(day, group_id):
 
     fetched_data = db.get_window_based_on_group(g.cursor, group_id, start_day,
                                                 end_day)
-
+    # Pop off first result because it corresponds to 00:00 of the next day
+    fetched_data = fetched_data[1:]
```

Comment:
yeah, that's a much better idea

On Fri Jan 30 2015 at 3:55:15 PM Nate <notifications@github.com> wrote:

> In density/density.py
> <https://github.com/adicu/density/pull/85#discussion_r23871693>:
>
> > @@ -213,7 +214,8 @@ def get_day_group_data(day, group_id):
> >
> >      fetched_data = db.get_window_based_on_group(g.cursor, group_id, start_day,
> >                                                  end_day)
> > -
> > +    # Pop off first result because it corresponds to 00:00 of the next day
> > +    fetched_data = fetched_data[1:]
>
> why not add 1 minute to the start time to skip querying the result
> altogether?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adicu/density/pull/85/files#r23871693>.
>